### PR TITLE
Clarify the description of the NULL argument in SSL_set1_host().

### PR DESCRIPTION
**Pull request by paulidale**
Moving a command to make the sentence parse better.

- [x] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: doc/man3/SSL_set1_host.pod [hunk 1]
```diff
@@ -19,8 +19,8 @@ SSL server verification parameters
 These functions configure server hostname checks in the SSL client.
 
 SSL_set1_host() sets the expected DNS hostname to B<name> clearing
-any previously specified hostname.  If B<name> is NULL,
-or the empty string the list of hostnames is cleared, and name
+any previously specified hostname.  If B<name> is NULL
+or the empty string, the list of hostnames is cleared, and name
```
Comment (mspncp) [inline doc/man3/SSL_set1_host.pod]: cleared and (remove comma)

Comment (paulidale) [reply doc/man3/SSL_set1_host.pod]: Done.

**Comment by levitte** [comment]
So wait, this just removes two commas???

**Comment by levitte** [comment]
Oh!  and adds

**Comment by paulidale** [comment]
Merged to master.  Thanks.

**Pull request closed without merging**